app/views/adventure_groups/_adventure_group.erb

```diff
@@ -0,0 +1,16 @@
+<% ratio = adventure_group.points.fdiv(max) %>
```

Comment:
Really cool way to solve this 😄 

Might be good to use some caching on these pages, but that can wait.